code/API_definitions/device-data-volume.yaml

```diff
@@ -24,13 +24,15 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: wip
+  version: 0.1.0-alpha.1
+  x-camara-commonalities: 0.5
+
 externalDocs:
   description: Product documentation at CAMARA
   url: https://github.com/camaraproject/
 
 servers:
-  - url: "{apiRoot}/device-data-volume/vwip"
+  - url: "{apiRoot}/device-data-volume/v0.1alpha1"
     variables:
       apiRoot:
         default: http://localhost:9092
```

Comment:
Is there a reason why this is 9092 and not 9091 as in the API guidelines? (not really important, but maybe for consistency)

Reply:
No reason - that was in the original PR from @VijayKesharwani

Personally, I'd prefer an HTTPS example, as ICM guidelines state TLS is mandatory. One day, I will open a PR in Commonalities for that.

Reply:
Seen that within Tenure ;-)